styles.css

```diff
@@ -1,785 +1,862 @@
+            height: 2.5vw;
+            background: #EBEBFF;
+            border-radius: 8px;
+            flex: none;
+            order: 2;
+            flex-grow: 0;
+          }
+          .cssbut{
+            width: 15%;
+            height: 4vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.25vw;
+            line-height: 4vw;
+            letter-spacing: 0.03em;
+            color: #6070FF;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            border: none;
+          }
+          .butjs{
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding: 4px 12px;
+            width: 42.5%;
+            height: 2.5vw;
+            background: #EBEBFF;
+            border-radius: 8px;
+            flex: none;
+            order: 3;
+            flex-grow: 0;
+          }
+          .jsbut{
+            width: 25%;
+            height: 4vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.25vw;
+            line-height: 4vw;
+            letter-spacing: 0.03em;
+            color: #6070FF;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            border: none;
+            padding-left: 0.5vw;
+          }
+          .rail{
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding: 4px 12px;
+            width: 69.2%;
+            height: 2.5vw;
+            background: #EBEBFF;
+            border-radius: 8px;
+            flex: none;
+            order: 1;
+            flex-grow: 0;
+          }
+          .railbut{
+            width: 75%;
+            height: 4vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.25vw;
+            line-height: 4vw;
+            letter-spacing: 0.03em;
+            color: #6070FF;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            border: none;
+            padding-left: 0.5vw;
+          }
+          .see{
+            width: 15.5vw;
+            height: 6vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.75vw;
+            line-height: 8vw;
+            display: flex;
+            align-items: center;
+            text-align: center;
+            padding-left: 2vw;
+            letter-spacing: 0.03em;
+            color: #396DF2;
+            flex: none;
+            order: 3;
+            flex-grow: 0;
+            border: #396DF2 solid 2px;
+            background-color: #FFFFFF;
+            border-radius: 8px;
+            margin-left: 5vw;
+            margin-top: 2vw;
+            cursor: pointer;
+          }

```

Comment:
- [ ] As I mentioned previously we have to stick to the Figma design as much as possible and the buttons in your design seem to be a bit out of place so please make sure they look similar to the Figma design.
- Figma design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/7b53cba8-d0cb-4cfc-8250-cae48963c0a1)
- Your design:  
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/ba2b0627-a69d-484f-9218-d4ec19e33d34)